services/thermal_mgr/thermal_mgr.c

```diff
@@ -42,19 +42,62 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 }
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
+  if (event == 0){
+    return ERR_CODE_INVALID_ARG;
+  }
+
   /* Send an event to the thermal manager queue */
+  if (xQueueSend(thermalMgrQueueHandle, event, (TickType_t)5) != pdPASS){
```

Comment:
- [ ] Check that the queue isn't null before use
- [ ] use 0 for the timeout as we don't want to bloc in isr